docs/en/reference/platforms.rst

```diff
@@ -17,7 +17,7 @@ instance by calling the ``getDatabasePlatform()`` method.
     $platform = $conn->getDatabasePlatform();
 
 Each database driver has a platform associated with it by default.
-Several drivers also share the same platform, for example PDO\_OCI
+Several drivers also share the same platform, for example ``PDO_OCI``
 and OCI8 share the ``OraclePlatform``.
```

Comment:
Should `OCI8` also be marked as a code block then?

Reply:
Yes, marked.